Makefile

```diff
@@ -0,0 +1,25 @@
+generate:
+	protoc \
+		-I versionpb \
+		-I third_party/grpc-gateway/ \
+		-I third_party/googleapis \
+		--go_out=plugins=grpc,paths=source_relative:./versionpb \
+		--grpc-gateway_out=./versionpb \
+		--openapiv2_out=third_party/OpenAPI/ \
+		versionpb/version.proto
```

Comment:
I think you are missing an opportunity to use [prototool](https://github.com/uber/prototool) there. It automatically passes correct flags to `protoc`, and can also format .proto files similar to `gofmt`, lint them, etc.

We use it several repositories, most recently in dbaas-api:
* https://github.com/percona-platform/dbaas-api/blob/main/Makefile
* https://github.com/percona-platform/dbaas-api/blob/main/Dockerfile
* https://github.com/percona-platform/dbaas-api/blob/main/api/prototool.yaml

Reply:
Or from PMM:
* https://github.com/percona/pmm/blob/PMM-2.0/api/Makefile
* https://github.com/percona/pmm/blob/PMM-2.0/api/prototool.yaml

Reply:
https://github.com/Percona-Lab/percona-version-service/pull/1/commits/9ac77a0ffa27c8fbfa93aebd17f784ac4c00c4fa